java/operator/src/main/java/com/instaclustr/cassandra/operator/controller/DataCenterDeletionController.java

```diff
@@ -106,6 +96,12 @@ public void deleteDataCenter() throws Exception {
                 }
             });
 
+
+            // delete persistent volumes & persistent volume claims
+            if(allowCleanups) {
```

Comment:
Minor nit: code style is space between `if` and opening bracket: eg: `if (allowCleanups) {`